person.rb

```diff
@@ -0,0 +1,20 @@
+class Person
+  attr_accessor :id, :name, :age
+
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_people.md#:~:text=and%20true.-,Getters%20for%20%40id,-%2C%20%40name%2C%20and), Please add only Getters for the `:id`, not Getters and setters as you did.